fmask/cmdline/sentinel2Stacked.py

```diff
@@ -54,7 +54,7 @@
     raise fmaskerrors.FmaskInstallationError(msg)
 
 
-def getCmdargs():
+def getCmdargs(argv=None):
     """
     Get command line arguments
     """
```

Comment:
shouldn't `argv` be passed to `parser.parse_args`?

Reply:
Yes, it should. Sorry, I modified online after checking in local and forget that last occurrence of argv.